controlplane/eks/api/v1alpha3/awsmanagedcontrolplane_types.go

```diff
@@ -153,6 +153,11 @@ type AWSManagedControlPlaneSpec struct { //nolint: maligned
 	// +optional
 	Addons *[]Addon `json:"addons,omitempty"`
 
+	// IdentityProviderconfig is used to specify the oidc provider config
+	// to be attached with this eks cluster
+	// +optional
+	OIDCIdentityProviderConfig *OIDCIdentityProviderConfig `json:"oidcIdentityProviderConfig,omitempty"`
```

Comment:
As this features is for v1alpha4 then the type declarations in v1alpha3 can be removed,